src/otel/OTelGroupCallMembership.ts

```diff
@@ -174,7 +176,7 @@ export class OTelGroupCallMembership {
     for (const [userId, userCalls] of calls.entries()) {
       for (const [deviceId, call] of userCalls.entries()) {
         if (!this.callsByCallId.has(call.callId)) {
-          const span = ElementCallOpenTelemetry.instance.tracer.startSpan(
+          const span = ElementCallOpenTelemetry.instance?.tracer.startSpan(
```

Comment:
We should add ` if (!ElementCallOpenTelemetry.instance) return;` here as well. Because `span` could be `undefined`. This isn't quite as obvious as `ElementCallOpenTelemetry.instance` doesn't have `undefined` in the signature.

Reply:
Good point. I've skipped the block rather than the whole function so we still remove calls from the active calls array.